randovania/games/super_metroid/logic_database/Maridia.txt

```diff
@@ -795,13 +826,27 @@ Extra - asset_id: 151
   * Layers: default
   * Normal Door to Aqueduct/Door to Botwoon Hallway
   > Door to Botwoon's Room
-      Gravity Suit and Speed Booster
+      Any of the following:
+          # With Speed Booster
+          Gravity Suit and Speed Booster
+          All of the following:
+              # Ice Clip
+              Ice Beam and Ice Beam Abuse (Intermediate)
+              # Damage to Survive the Mocktroids
+              Normal Damage ≥ 100
```

Comment:
is doing the mocktroids hitless really that difficult?

Reply:
The ice clip requires you to take damage from the mocktroids.